src/pages/settings/index.tsx

```diff
@@ -1,6 +1,26 @@
 import Head from "next/head"
+import { styled } from "linaria/react"
 
 import TopBar from "../../components/TopBar"
+import SettingItem from "../../components/SettingItem"
+
+const Main = styled.main`
```

Comment:
The entire window except the tab bar is contained within a main so don't wanna nest a main within a main, for this I think an `<article />` is recommended